src/components/files/icons.jsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+// TODO: make svg icons and use SvgIcon as in https://v0.material-ui.com/#/components/svg-icon
+const GithubIcon = () => <img src='http://localhost:8000/static/icons/github-circle.png' height="48" width="48" alt=""/>;
+const SciGymIcon = () => <img src='http://localhost:8000/static/icons/scigym-logo.png' height="37" width="37" alt="" style={{padding: "5px"}}/>
+
+export {GithubIcon, SciGymIcon} 
```

Comment:
you can do:

````
export const GithubIcon = () => ...
export const SciGymIcon = () => ...
```
and leave off the bottom export. It's a little easier to see inline that it's an exported variable.